lecture/src/test/java/de/tum/in/www1/artemis/lecture/IntegrationTest.java

```diff
@@ -0,0 +1,17 @@
+package de.tum.in.www1.artemis.lecture;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import org.springframework.boot.test.context.SpringBootTest;
+
+/**
+ * Base composite annotation for integration tests.
+ */
+@Target(ElementType.TYPE)
```

Comment:
Just as preparation or can this be removed?

Reply:
it's used by ExceptionTranslatorTest

Reply:
Ah, I missed the fact that it's an annotation. Right.